src/features/llms/runtime/server/routes/llms_full.txt.get.ts

```diff
@@ -0,0 +1,65 @@
+import { joinURL, hasProtocol } from 'ufo'
+import { eventHandler, setHeader } from 'h3'
+import type { MinimalTree } from '@nuxt/content'
+import { decompressTree } from '@nuxt/content/runtime'
+import type { MDCElement } from '@nuxtjs/mdc'
+import type { LLMSOptions } from '../../types'
+import { useRuntimeConfig } from '#imports'
+
+const linkProps = ['href', 'src', 'to']
+
+export default eventHandler(async (event) => {
+  const stringifyMarkdown = await import('@nuxtjs/mdc/runtime').then(res => res.stringifyMarkdown)
+  const visit = await import('unist-util-visit').then(res => res.visit)
```

Comment:
Try to exclude imports from the server's entry point using dynamic import.

However, it does not work as expected and imports are listed in Nitro's server entry. I'm not sure about the root cause.

/cc @danielroe @pi0